ls-ecaps.c

```diff
@@ -653,8 +653,9 @@ cap_rcec(struct device *d, int where)
       int prevmatched=0;
       int adjcount=0;
       int prevdev=0;
+      int dev;
       printf("RCiEP at Device(s):");
-      for (int dev=0; dev < 32; dev++)
+      for (dev=0; dev < 32; dev++)
```

Comment:
The error message suggests that the compiler supports C99 mode, so turn it on please.

Years ago, I preferred to avoid C99 in the pciutils, but that times are long gone. It's 22 years since C99 was released, so it's time to say farewall to compilers which still do not support it.

Reply:
Ah, stupid that it is disabled by default... With `CFLAGS="-std=gnu99"` this one patch is not needed.